target_frame: velodyne_link # Leave disabled to output scan in pointcloud frame
transform_tolerance: 0.01
min_height: -0.8 #-0.7 #-0.4
max_height: 0.3 #1.5
angle_min: -2.6178 #-2.4435 #-2.3565
angle_max: 2.6178 #2.4435 #2.3565
angle_increment: 0.008711645
scan_time: 0.03333
range_min: 0.2
range_max: 100.0
use_inf: true

# Concurrency level, affects number of pointclouds queued for processing and number of threads used
# 0 : Detect number of cores
# 1 : Single threaded
# 2->inf : Parallelism level
concurrency_level: 1
